modules/web/client/src/main/publicdev/environments.conf.json

```diff
@@ -2,7 +2,7 @@
   {
     "hostName": "*",
     "environment": "DEVELOPMENT",
-    "odbURI": "wss://lucuma-postgres-odb-dev.herokuapp.com/ws",
+    "odbURI": "ws://localhost:8082/ws",
```

Comment:
Do we need to run a local. odb?

Reply:
This slipped through, I'll revert. We have logic in place to pick up `local.conf.json` if need be.

Reply:
Actually, this shouldn't be checked in, it got out of sync with `.gitignore` with the directory change.